ambry-store/src/main/java/com/github/ambry/store/StorageManager.java

```diff
@@ -349,6 +349,14 @@ public boolean addBlobStore(ReplicaId replica) {
     return true;
   }
 
+  /**
+   * Helper utility to add replicaId to partitionToDiskManager.
+   * @param replicaId the {@link ReplicaId} whose partition and disk to use.
+   */
+  protected void updatePartitionToDiskManager(ReplicaId replicaId) {
+    partitionToDiskManager.put(replicaId.getPartitionId(), diskToDiskManager.get(replicaId.getDiskId()));
```

Comment:
Note: added because modified line in MockStorageManager hits a compile error in JDK11 trying to access package private class Disk.

Reply:
nit: [optional]  probably move this method to somewhere after public methods.